devutils/update_lists.py

```diff
@@ -74,6 +75,7 @@
     '*.woff',
     '*.woff2',
     '*makefile',
+    '*.profdata',
```

Comment:
So, profdata will be permanently excluded unlike how it is now?

Reply:
Yes, keeping it in the pruning list would require the pgo archive to be downloaded and extracted, only to later delete the file.  Not to mention that some people want to utilize pgo profiles in their builds and have been manually exempting the file already.